src/admin/routes.js

```diff
@@ -65,17 +67,6 @@ export const parse = (route, template) => {
     }, {});
 };
 
-export const getSingleQueryString = (key, value) => (Array.isArray(value) ?
-    value.map(item => `${encodeURIComponent(key)}=${encodeURIComponent(item)}`).join('&') :
-    `${encodeURIComponent(key)}=${encodeURIComponent(value)}`);
-
-// creates a query string from a plain js object
-export const queryString = query => (
-    query ?
-        `?${Object.keys(query).map(k => getSingleQueryString(k, query[k])).join('&')}` :
-        ''
-);
-
```

Comment:
I had to use this function as well, so I refactored it into 'utils', and added tests.

But I particularly dont like that for queryString({}) it gives '?' -- would you think its wise to change it and modify according code?